test/Util.java

```diff
@@ -54,7 +54,7 @@ public class Util {
 
     public static File getArchivePath(int index) {
         String[] args = System.getProperty("sun.java.command").split(" ");
-        if (args.length < index) {
+        if (args.length <= index) {
```

Comment:
I think if we go with `if (index < args.length)` it would read better. What do you think?

Reply:
If we make it `if (!(index < args.length))` then the rest of the code can stay the same. Otherwise, we would have to follow a pattern where the 'happy' path enters the if statement scope. Does that work stylistically?

Reply:
Yeah you're right!

In that case then maybe `if (index >= args.length)`? Basically your first solution and we just revert `index` and `args.length`. It looks simpler than `if (!(index < args.length))`.

Reply:
Agreed. Ready to merge after CI.